docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -411,13 +441,19 @@ PublishModeQosPolicy
 This QoS Policy configures how the |DataWriter| sends the data.
 See |PublishModeQosPolicy-api|.
 
+It also configures the name of the flow controller to use when asynchronous publishing is used.
+It should be the name of a flow controller registered on the creation of the DomainParticipant.
+See |FlowControllersQos|.
```

Comment:
For this link to properly work, it has to be defined in `03-exports/aliases.include`:

```
|FlowControllersQos| replace:: :ref:`FlowControllersQos<flowcontrollersqos>`